1.16.5/src/main/java/io/github/axolotlclient/mixin/PlayerListHudMixin.java

```diff
@@ -262,7 +262,7 @@ public abstract class PlayerListHudMixin {
 		if (player == null) {
 			return;
 		}
-		cir.setReturnValue(player.getTabListDisplay());
+		cir.setReturnValue(Text.of(player.getTabListDisplay()));
```

Comment:
There is no Text.of() constructor in 1.16 yet, use `new LiteralText()` instead.

Reply:
Unless I am missing something, `Text.of` works on 1.16. It compiles and fixed the issue for me, and it is in the [javadocs](https://maven.fabricmc.net/docs/yarn-1.16.5+build.2/net/minecraft/text/Text.html).

Reply:
Seems to build.